linode/manager
# change: [M3-7583] - Update `react-waypoint` for React 18

## Description 📝

- Updates `react-waypoint` to `latest` (M3-7583) in preparation for **React 18** (M3-6199)

## Preview 📷

https://github.com/linode/manager/assets/115251059/c8117770-389b-4ed4-a9f3-33469ee17d03

## How to test 🧪

- To confirm that this update didn't break anything, test **infinite scrolling**
- All of the places in Cloud Manager that use `react-waypoint` ⬇️
  - Object Storage bucket with many objects
  - Events Landing with many events
  - Public StackScripts
  - Support Ticket with many comments
  - AGLB with many configurations
  - LKE Cluster with many Node Pools

## As an Author I have considered 🤔
